hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -189,7 +189,11 @@ public static String getCommitTime(String fullFileName) {
   }
 
   public static long getFileSize(FileSystem fs, Path path) throws IOException {
-    return fs.getFileStatus(path).getLen();
+    if (fs.exists(path)) {
+      return fs.getFileStatus(path).getLen();
+    } else {
+      return 0L;
+    }
```

Comment:
What is the context here ? Can we ensure the file must exist when calling the `getFileSize`